### DISCUSSION TITLE
Need help with integration in iOS

### DISCUSSION THREAD

**Discussion by 11Shraddha** [readium/mobile#19] [Q&A]
Hello I have got all the library downloaded. But do not know how to integrate it in my project. Can anyone please share the steps how to add it in my current iOS project?

Thanks in advance.

**Comment by mickael-menu** [comment]
You can take a look at the workspace repository showing different ways of integrating the libraries: https://github.com/readium/r2-workspace-swift

And the test app showcases how to use them https://github.com/readium/r2-testapp-swift
